### ISSUE TITLE
Syntax error on UPDATE

### ISSUE THREAD

**Issue by Mexican-Man** [G4brym/workers-qb#5]
I have a basic "update multiple fields" query that throws a syntax error. It's generating this query:
```sql
UPDATE pages SET (content = ?2) WHERE id = ?1
```
–which doesn't appear to be a valid statement. I'm not super familiar with SQL anymore, but it seems like removing the parentheses would correct the syntax.

Additionally, looking through the code, I noticed this query:
```sql
UPDATE pages SET (content = ?2, category = ?3) WHERE id = ?1
```
–was being called with these parameters being binded:
```js
[ 'banana1', 'news', 'test' ]
```
In this case, the id is `test`, the content is `banana1` and the category is `news`. It seems like `?1` and `?2` are being swapped. Is this intentional?

**Comment by G4brym** [comment]
Hey @Mexican-Man thanks for reporting this issue
I will take a look into this next week, then I will report back to this issue when it's solved

**Comment by G4brym** [comment]
Thanks for reporting this issues, I've just fixed them
I will be pushing a new release today or tomorrow with this

Let me know if you find any more issues or things that could be improved

**Comment by G4brym** [comment]
I've just published a new version v0.1.3 with this fix
